Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -266,6 +272,34 @@ void DMainWindow::OnPlay()
 	}
 }
 
+// Toggles fullscreen
+// Game assumed running
```

Comment:
OK, well, remove the first comment, and make the second a bit less abbreviated then.